empty/database/db.js

```diff
@@ -0,0 +1,23 @@
+var knex = require('./knex.js')
```

Comment:
Unless you are obligated to not ues ES6, don't use `var` - it has the potential to introduce subtle bugs... Use `let` or `const` when available, preferring `const` if the variable is not going to change.